R/geom-linerange.r

```diff
@@ -113,6 +113,7 @@ GeomLinerange <- ggproto("GeomLinerange", Geom,
   },
 
   draw_panel = function(data, panel_params, coord, flipped_aes = FALSE) {
+    browser()
```

Comment:
This looks left over. That's probably why the CI fails.

Reply:
Oh shit — thanks😅